torchaudio/datasets/librispeech.py

```diff
@@ -67,7 +67,7 @@ def load_librispeech_item(
             raise FileNotFoundError(f"Translation not found for {fileid_audio}")
 
     return (
-        waveform,
+        filepath,
```

Comment:
should we return the relative file path to the provided path/root, or the full file path (including path/root) that can directly be passed into `torchaudio.load()`

Reply:
I vote for relative path, because that will allow to iterate on dataset exists in places other than local file system.